locales/de-de.json

```diff
@@ -0,0 +1,74 @@
+	"driver-order-new.call_to_action": "Bitte wähle eine Aktion 👇 oder ignoriere diese Mitteilung.",
+	"driver-order-new.send_my_number": "Meine Nummer senden",
+	"driver-order-new.set_my_price": "Meinen Preis festlegen",
+	"driver-order-new.price_not_set": "Der Preis für diesen Auftrag wurde nicht vom Passagier festgelegt. Du musst deinen Preis angeben.",
+	"metric-distance.km": "%s km",
+	"passenger-request-price.set_price": "In dieser App setzt der Fahrgast den Preis fest 😉 Gebe uns deinen Preis an (in lokaler Währung). Wie viel bist du bereit zu zahlen? (Oder gebe \"0\" (Null) an, wenn du es nicht weisst)",
+	"passenger-request-price.should_be_numeric": "Hoppla, der Preis sollte der numerische Wert (und in lokaler Währung) sein! Zum Beispiel: \"50\". Bitte versuche es erneut!",
+	"passenger-request-price.on_timeout": "Es scheint du wartest schon zu lange? Das tut uns leid. Wenn du keine Fahrt gefunden hast, empfehlen wir es später erneut zu versuchen.",
+	"passenger-contact-new-number.message": "🔔 {{driver}} akzeptiert deinen Preis! Entfernung zum Fahrer  ~{{distance}}. Rufe den Fahrer an um deinen Auftrag zu bestätigen:",
+	"driver-order-set-price.set_price": "Lege deinen Preis fest (in lokaler Währung). Du kannst Details einfügen, die deinen Preis erläutern. Zum Beispiel: \"20, wegen der Brückenreparatur\" oder \"15, wenn der Verkehr in Ordnung ist\"",
+	"passenger-contact-driver-price.message": "🔔 ⚠️ {{driver}} bietet einen anderen Preis: \"{{price}}\". Entfernung zum Fahrer ~{{distance}}. Wenn du mit dem Preis einverstanden bist, rufe den Fahrer an um deinen Auftrag zu bestätigen:",
+	"passenger-verify-location.verify_location": "Bitte schaue in die Karte oben (zum Vergrößern klicken), ist dieser Standort korrekt?",
+	"passenger-verify-location.try_with_paperclip": "Bitte versuche deine Standort zu senden durch Tippen auf die Büroklammer, und danach auf das Standort-Icon.",
+	"passenger-verify-cash.verify_cash": "Die meisten unserer Fahrer akzeptieren keine Kreditkarten. Haste du Bargeld?",
+	"passenger-verify-cash.get_some_cash": "Leider musst du Bargeld haben, bevor du mitfahren kannst. Versuche es erneut wenn du Bargeld hast!",
+	"driver-request-radius.request_radius": "Bis zu welcher Entfernung vom Passagier bist du an Aufträgen interessiert? Wähle von der Liste 👇 or gebe eine Zahl ein (bis zu 10 km).",
+	"driver-request-radius.option1": "1 km",
+	"driver-request-radius.option3": "3 km, Empfohlen",
+	"driver-request-radius.option5": "5 km",
+	"driver-request-radius.should_be_valid": "Kann deine Wahl nicht verstehen. Wähle die Option 👇 oder gebe eine Zahl von 1 bis 10 ein.",
+	"global.location_button_text": "Standort senden",
+	"global.default_options_message": "Bitte wähle eine Aktion aus dem Menü unten 👇",
+	"global.default_inline_options_message": "Klicke aus die Taste 👇 oder ignoriere diese Mitteilung.",
+	"global.replied_to_order": "✔️ Du hast auf diesen ☝️ Auftrag geantwortet. Erwarte einen Anruf vom Passagier um zu bestätigen.",
```

Comment:
How about:

Du hast auf diesen Auftrag geantwortet und wirst in Kürze einen Anruf vom Passagier erhalten um den Auftrag zu bestätigen.